pkgs/top-level/perl-packages.nix

```diff
@@ -6116,7 +6128,7 @@ let
       sha256 = "c065fcd3e2f22ae769937bcc971b91f80294d5009fac140bfba83bf7d35305e3";
     };
     configurePhase = ''
```

Comment:
This probably should better be ```buildPhase```, it is ```configurePhase``` because if was impossible to override ```buildPhase```.
I kept it as it for a while.